apps/arweave/src/ar_http_iface_client.erl

```diff
@@ -1255,7 +1255,8 @@ get_peers(Peer) ->
 					timeout => 2 * 1000
 				}),
 			PeerArray = ar_serialize:dejsonify(Body),
-			lists:map(fun ar_util:parse_peer/1, PeerArray)
+			PeersList = lists:map(fun ar_util:parse_peer/1, PeerArray),
```

Comment:
Why parse_peer here rather than safe_parse_peer?

Reply:
Oop never mind. I miss read the changes. I though you introduced the safe_parse_peer as a wrapper around parse_peer.